File: 06_numerosParesImpares/Csharp/parImpar.cs
using System;
public class parImpar
{
    public static void Main(String[] args)
    {
        int num;
        Console.Write("Ingresa un numero: ");
        num=Convert.ToInt32(Console.ReadLine());
        if(num%2==0)
            Console.WriteLine("{0} es par",num);
        else
            Console.WriteLine("{0} es impar",num);
    }
}

File: 06_numerosParesImpares/C/parImpar.c
#include<stdio.h>
int num;
int main()
{
    printf("Ingresa numero: ");
    scanf("%d",&num);
    if(num%2==0)
        printf("%d es par\n",num);
    else
        printf("%d es impar\n", num);
    return 0;
}